examples/benchmarks/bubblebench/ot/src/main.ts

```diff
@@ -17,7 +17,7 @@ export class Bubblebench extends DataObject {
 	private maybeTree?: SharedJson1 = undefined;
 	private maybeAppState?: AppState = undefined;
 
-	protected async initializingFirstTime() {
+	protected async initializingFirstTime(): Promise<void> {
```

Comment:
I see you added a `Promise<void>` return type but I don't see the method returning a promise, it looks to be just a `void` - can you help me understand?

Reply:
Its signature is async, so it _must_ return a promise. And that signature is dictated from some superclass.